### PR TITLE
Add ;;math command

### PR DESCRIPTION
**Pull request by RealKC**
This adds a `;;math` command which can perform the next operations: addition, subtraction, multiplication, division, modulo, square root and can calculate the power of num1 to num2 and percentages.

This depends on a Fredboat-i13n PR I need to make.

Reviews would be nice.

Also yeah I fucked accounts again, GG me.

### CODE REVIEW

**Comment by RealKC** [comment]
Yeah yeah fuck off bot

**Comment by RealKC** [comment]
Reviews are welcome by the way.

**Comment by napstr** [comment]
Please fix the compile error.

image

**Comment by RealKC** [comment]
@napstr can check it out rn, build should fail because of the FredBoat-I13n PR

File: FredBoat/src/main/java/fredboat/command/util/MathCommand.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+                        return;
+                }
+
+            } else {
+                HelpCommand.sendFormattedCommandHelp(context);
+                return;
+            }
+
+        } catch(NumberFormatException ex) {
+            HelpCommand.sendFormattedCommandHelp(context);
```
Comment (napstr) [inline FredBoat/src/main/java/fredboat/command/util/MathCommand.java]: Should think about a more specific message here. "could not parse at least one of you numbers" or somethign like that.

**Comment by RealKC** [comment]
Oh I forgot to add the #, lemme add them.

Personally not sure how to remove the trailing 0s in numbers like that.

File: FredBoat/src/main/java/fredboat/command/util/MathCommand.java (hunk 1 above)
Comment (RealKC) [reply FredBoat/src/main/java/fredboat/command/util/MathCommand.java]: Can't think of a better message, so I'll use that(unless someone has better suggestions).

**Comment by Shredder121** [comment]
trimTrailingZeros?
It by default takes the same significant digits as the input arguments

**Pull request merged**